### PR TITLE
feat: Add no-k8s image.

### PR DESCRIPTION
**Pull request by gary-van-woerkens**

### CODE REVIEW

**Comment by revolunet** [comment]
Est-ce que le Dockerfile ne devrait pas etre un truc comme ca ?

```Dockerfile
FROM registry.gitlab.factory.social.gouv.fr/socialgouv/docker/kosko:4.2.0

RUN apk add --update --no-cache git=~2

# commentaire pour que renovate maintienne à jour
RUN git clone --depth 1  --branch vx.y.z https://github.com/SocialGouv/.k8s .

RUN cd .k8s; yarn install --frozen-lockfile; cd -
```

File: .gitignore [hunk 1]
```diff
@@ -5,4 +5,4 @@
 
 # misc
 .DS_Store
-
+.vscode
```
Comment (douglasduteil) [inline .gitignore:8]: Should be a global gitignore has not directly related to this project

File: no-k8s/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,8 @@
+FROM registry.gitlab.factory.social.gouv.fr/socialgouv/docker/kosko:4.2.0
+
+RUN apk add --update --no-cache git=~2
+
+# renovate: datasource=github-tags lookupName=socialgouv/.k8s versioning=1.0.1
+RUN git clone --depth 1 --branch v1.0.1 https://github.com/SocialGouv/.k8s /.k8s
```
Comment (douglasduteil) [inline no-k8s/Dockerfile:6]: Looks like one of our [templates](https://github.com/SocialGouv/kosko-charts/tree/master/templates)
We should move those files to `SocialGouv/kosko-chart`

**Review by douglasduteil — CHANGES_REQUESTED** [review]
Good idea @gary-van-woerkens

File: no-k8s/Dockerfile (hunk 2 above)
Comment (douglasduteil) [reply no-k8s/Dockerfile:6]: Even this image can be generated in the `SocialGouv/kosko-chart` repo 🤔

File: no-k8s/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+
+RUN cd .k8s; yarn install --frozen-lockfile; cd -
```
Comment (douglasduteil) [inline no-k8s/Dockerfile:8]: We might want to have a production ready version with no extra cache by default

```suggestion
WORKDIR /.k8s
RUN yarn install --frozen-lockfile --production --cache-folder /dev/shm/yarn
```

File: .gitignore (hunk 1 above)
Comment (gary-van-woerkens) [reply .gitignore:8]: What's the difference between `.vscode` and `.DS_Store` in this case ?

File: no-k8s/Dockerfile (hunk 2 above)
Comment (revolunet) [reply no-k8s/Dockerfile]: should we really mix kosko-charts (the lib) releases with this template (user-land) releases ?

File: .gitignore (hunk 1 above)
Comment (douglasduteil) [reply .gitignore]: non go global gitignore

File: no-k8s/Dockerfile (hunk 2 above)
Comment (douglasduteil) [reply no-k8s/Dockerfile]: Isn't it a one of the template of the lib ?

File: no-k8s/Dockerfile (hunk 3 above)
Comment (gary-van-woerkens) [reply no-k8s/Dockerfile]: Done.

File: .gitignore (hunk 1 above)
Comment (gary-van-woerkens) [reply .gitignore]: Alright. Should I remove the entire file though?

File: no-k8s/Dockerfile (hunk 2 above)
Comment (gary-van-woerkens) [reply no-k8s/Dockerfile]: It looks like so, true. But this one might become a bit more "dynamic".
Let's keep it the way it is at the moment. We'll probably move this `.k8s` folder within `SocialGouv/kosko-chart` a bit later.

File: .gitignore (hunk 1 above)
Comment (gary-van-woerkens) [reply .gitignore]: Done.

**Review by douglasduteil — APPROVED** [review]

**Comment by douglasduteil** [comment]
Go :)

**Pull request merged**